<launch>
    <node pkg="boundary_estimation" type="boundary_estimation.py" name="boundary_estimation" respawn="true" output="screen">
        <remap from="/input/local_map" to="/ugr/car/map/fastslam/local"/>
        
        <remap from="/output/debug/left_boundary" to="/boundary_estimation/debug/left_boundary"/>
        <remap from="/output/debug/right_boundary" to="/boundary_estimation/debug/right_boundary"/>
        <remap from="/output/boundaries" to="/boundary_estimation/boundaries"/>

        <param name="max_distance" value="36" />
        <param name="min_distance" value="1" />
        <param name="max_angle_change" value="1.8" />
    </node>
</launch>